sysbox-in-docker/README.md

```diff
@@ -0,0 +1,91 @@
+[doc](../docs/distro-compat.md).
+
+* Root-privileges: As a container runtime, Sysbox requires root privileges to
+operate. As a result, Sysbox-In-Docker container must be launched in "privileged"
+mode.
+
+    **Note**: Within the privileged container, inner containers launched with Docker +
+    Sysbox will be strongly isolated from the host by Sysbox (e.g., via the Linux
+    user-namespace).
+
+* Required bind-mounts:
+
+    - "/var/lib/docker"
+
+    - "/var/lib/sysbox"
+
+    - "/lib/modules/<kernel>"
+
+    - "/usr/src/linux-headers-<kernel>"
+
+
+## Execution
+
+Sysbox-In-Docker environment can be created in three simple steps:
+
+1) Pick the Linux distribution on which to run Sysbox by choosing one of the
+Makefile targets displayed below.
+
+    ```
+    $ make sysbox-in-docker
+    ...
+    Usage:
+    make sysbox-in-docker <distro-release>
+
+    help                       Show supported docker images
+    centos-8                   Build CentOS-8 docker image
+    debian-buster              Build Debian-Buster docker image
+    debian-bullseye            Build Debian-Bullseye docker image
+    fedora-31                  Build Fedora-31 docker image
+    fedora-32                  Build Fedora-32 docker image
+    ubuntu-bionic              Build Ubuntu-Bionic docker image
+    ubuntu-focal               Build Ubuntu-Focal docker image
+    $
+    ```
+
+2) Build the Sysbox-In-Docker image by executing the chosen target. Notice that
+the precise instruction to utlize to launch the Sysbox-In-Docker container will
+be displayed as part of this output.
+
+    ```
+    $ make sysbox-in-docker ubuntu-bionic
+    ...
+    *** Launch container with the following instruction ***
+
+    docker run -it --privileged --rm --hostname sysbox-in-docker --name sysbox-in-docker -v /var/tmp/sysbox-var-lib-docker:/var/lib/docker -v /var/tmp/sysbox-var-lib-sysbox:/var/lib/sysbox -v /lib/modules/5.4.0-48-generic:/lib/modules/5.4.0-48-generic:ro -v /usr/src/linux-headers-5.4.0-48-generic:/usr/src/linux-headers-5.4.0-48-generic:ro -v /usr/src/linux-headers-5.4.0-48:/usr/src/linux-headers-5.4.0-48:ro nestybox/sysbox-in-docker:ubuntu-bionic
+
+    make: Nothing to be done for 'ubuntu-bionic'.
+    $
+    ```
+
+3) Create a Sysbox-In-Docker container by making use of the instruction obtained
+in the previous step:
+
+    ```
+    $ docker run -it --privileged --rm --hostname sysbox-in-docker --name sysbox-in-docker -v /var/tmp/sysbox-var-lib-docker:/var/lib/docker -v /var/tmp/sysbox-var-lib-sysbox:/var/lib/sysbox -v /lib/modules/5.4.0-48-generic:/lib/modules/5.4.0-48-generic:ro -v /usr/src/linux-headers-5.4.0-48-generic:/usr/src/linux-headers-5.4.0-48-generic:ro -v /usr/src/linux-headers-5.4.0-48:/usr/src/linux-headers-5.4.0-48:ro nestybox/sysbox-in-docker:ubuntu-bionic
+    ...
+    ```
+
+    In a separate terminal do ...
+
+    ```
+   $ docker exec -it sysbox-in-docker bash
+   #
+    ```
```

Comment:
Need more here. For example:

Inside the sysbox-in-docker container, you should see Docker and Sysbox running:

```
ps -fu root | egrep "sysbox|docker"
```

And you can now launch containers with Docker + Sysbox as usual:

```
# docker run  --runtime=sysbox-runc -it --rm nestybox/ubuntu-focal-systemd-docker
```

The containers launched by Sysbox are strongly isolated via the Linux user namespace and are capable of running not just microservices, but low-level system software such as systemd, Docker, K8s, and more.

Refer to the Sysbox quickstart guide for more examples on how to use Sysbox

Reply:
sure

Reply:
LG.